.github/workflows/pre-release-nethermind.yml

```diff
@@ -0,0 +1,153 @@
+      run: |
+        echo "PACKAGE_PREFIX=$PACKAGE_PREFIX" >> $GITHUB_OUTPUT
+        $SCRIPTS_PATH/archive-packages.sh
+    - name: Cache packages
+      uses: actions/cache@v3
+      with:
+        path: |
+          ${{ github.workspace }}/${{ env.PACKAGE_DIR }}/
+          ${{ env.SCRIPTS_PATH }}/
+        key: packages-${{ steps.build-runner.outputs.COMMIT_HASH }}
+
+  trigger-publish:
+    name: Trigger publish event to different sources
+    runs-on: ubuntu-latest
+    needs: build
+    environment:
+      name: Releases
+      url: https://github.com/NethermindEth/nethermind/releases/tag/${{ github.event.inputs.tag }}
+    steps:
+    - run: echo "Just a middle-man job"
+
+  publish-github:
+    name: Publish packages to GitHub Releases
+    runs-on: ubuntu-latest
+    needs: trigger-publish
+    steps:
+    - name: Restore packages from cache
+      uses: actions/cache@v3
+      with:
+        path: |
+          ${{ github.workspace }}/${{ env.PACKAGE_DIR }}/
+          ${{ env.SCRIPTS_PATH }}/
+        key: packages-${{ needs.build.outputs.COMMIT_HASH }}
+    - name: Publish
+      env:
+        GIT_COMMIT: ${{ needs.build.outputs.COMMIT_HASH }}
+        GIT_TAG: ${{ github.event.inputs.tag }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        PACKAGE_PREFIX: ${{ needs.build.outputs.PACKAGE_PREFIX }}
+      run: $SCRIPTS_PATH/publish-github.sh
+
+  publish-dockers:
+    name: Publish Docker images to Docker Hub
+    runs-on: ubuntu-latest
+    needs: trigger-publish
+    env:
+      DOCKER_IMAGE: nethermind/nethermind
+    steps:
+    - name: Check out Nethermind repository
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.inputs.tag }}
+        fetch-depth: 0
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v2
+    - name: Log in to Docker Hub
+      if: success()
+      run: |
+        echo "${{ secrets.DOCKER_ACCESS_TOKEN }}" | docker login --username "${{ secrets.DOCKER_USERNAME }}" --password-stdin
+    - name: Build and push image to Docker registry (major) / trigger DAppNode build
+      if: ${{ !contains(github.event.inputs.tag, '-') }}
+      run: |
+        docker buildx build --platform=linux/amd64,linux/arm64 -t "${{ env.DOCKER_IMAGE }}:latest" -t "${{ env.DOCKER_IMAGE }}:${{ github.event.inputs.tag }}" -f Dockerfile --build-arg COMMIT_HASH=${{ needs.build.outputs.COMMIT_HASH }} --build-arg BUILD_TIMESTAMP=${{ needs.build.outputs.BUILD_TIMESTAMP }} . --push
+        curl -s -X POST -u "${{ secrets.REPOSITORY_DISPATCH_TOKEN }}" -H "Accept: application/vnd.github.everest-preview+json" -H "Content-Type: application/json" -d '{"event_type":"dappnode","client_payload":{"tag":"${{ github.event.inputs.tag }}"}}' https://api.github.com/repos/$GITHUB_REPOSITORY/dispatches
+    - name: Build and push image to Docker registry  (patch)
```

Comment:
Let's remove this as we are already pushing it above. (to "unstable" and to RC version number)